nextcord/abc.py

```diff
@@ -523,6 +533,14 @@ def permissions_synced(self) -> bool:
         category = self.guild.get_channel(self.category_id)
         return bool(category and category.overwrites == self.overwrites)
 
+    @property
+    def jump_url(self) -> str:
```

Comment:
I'm not opposed to it being in `CategoryChannel`, since it's valid. If others are happy, I'll leave it in there (didn't realise I put it in there in the first place, lol).